napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -1,21 +1,19 @@
 from qtpy.QtCore import Qt
-from qtpy.QtWidgets import QComboBox, QFrame, QGridLayout
+from qtpy.QtWidgets import QComboBox, QFormLayout, QFrame
 
 from ...layers.base._base_constants import BLENDING_TRANSLATIONS
 from ...utils.events import disconnect_events
 from ..widgets._slider_compat import QDoubleSlider
 
 
-class LayerListGridLayout(QGridLayout):
+class LayerGridLayout(QFormLayout):
```

Comment:
We don't have to hold up the PR for this, but maybe just noting that this could now be `LayerFormLayout` instead and doc string could call out `form` instead of `grid`